keanu-project/src/main/java/io/improbable/keanu/util/dot/VertexDotLabel.java

```diff
@@ -0,0 +1,66 @@
+package io.improbable.keanu.util.dot;
+
+import io.improbable.keanu.vertices.ConstantVertex;
+import io.improbable.keanu.vertices.Vertex;
+
+import java.util.Objects;
+
+public class VertexDotLabel {
+    private Vertex vertex;
+    private String value = "";
+    private String annotation = "";
+    private String vertexLabel = "";
+    private static final String DOT_LABEL_OPENING = "[label=\"";
+    private static final String DOT_LABEL_CLOSING = "\"]";
+
+    public VertexDotLabel(Vertex vertex) {
+        this.vertex = vertex;
+    }
+
+    public enum VertexDotLabelType{
+        VALUE, CLASS_NAME, ANNOTATION_LABEL, VERTEX_LABEL
+    }
+
+    public void setDotLabel(VertexDotLabelType labelType, String label) {
+        switch (labelType) {
+            case VALUE:
+                value = label;
+                break;
+            case VERTEX_LABEL:
+                vertexLabel = label;
+                break;
+            case ANNOTATION_LABEL:
+                annotation = label;
+        }
```

Comment:
That's because it's never intended to be used to set class name (removed it from the enum), and it should not be possible to pass anything else than the options covered in. Should I add a default that throws an exception in that case?

Reply:
(removed the method and enums)